server/index.js

```diff
@@ -0,0 +1,26 @@
+const express = require('express');
+const app = express();
+const helperFunc = require('../database/helper.js')
```

Comment:
This is technically an object rather than a function - I recommend changing the name to be more descriptive, or just grabbing the function directly. For example, you could call this `db` and then use `db.getReviews()` below, or you could grab just the getReviews function and use it directly like `const getReviews = require('../database/helper.js').getReviews` or `const { getReviews } = require('../database/helpers.js')`

Reply:
Ill make these changes today. Thanks for the review Chris!